tests/com/google/common/css/compiler/gssfunctions/GssFunctionsTest.java

```diff
@@ -204,6 +204,96 @@ public void testMakeTranslucent_otherLocale() {
     }
   }
 
+  public void testSaturateColor() throws GssFunctionException {
+    GssFunctions.SaturateColor function = new GssFunctions.SaturateColor();
+    assertEquals("#2F5BD8",
+        function.getCallResultString(ImmutableList.of("#5a7bd8", "20")));
+  }
+
+  public void testSaturateColor_otherLocale() throws GssFunctionException {
+    Locale.setDefault(Locale.FRANCE);
+    try {
+
+      GssFunctions.SaturateColor function = new GssFunctions.SaturateColor();
+      assertEquals("#2F5BD8",
+        function.getCallResultString(ImmutableList.of("#5a7bd8", "20")));
```

Comment:
The `_otherLocale` tests only matter if there is a fractional part to the result. See issue #14.